alphalens/plotting.py

```diff
@@ -724,6 +724,11 @@ def plot_cumulative_returns(factor_returns, period, title=None, ax=None):
         Length of period for which the returns are computed (e.g. 1 day)
         if 'period' is a string it must follow pandas.Timedelta constructor
         format (e.g. '1 days', '1D', '30m', '3h', '1D1h', etc)
+    freq : pandas DateOffset
+        Used to specify a particular trading calendar e.g. BusinessDay or Day
+        Usually this is inferred from utils.infer_trading_calendar, which is
```

Comment:
If it's inferred, why is it a required argument?

Reply:
Because `plot_cumulative_returns` doesn't receive `factor_data` in input, which has the inferred trading calendar.